docs/content/en/docs/getting_started.md

```diff
@@ -12,16 +12,12 @@ weight: 1
     `>=5.5`
 -   A C compiler (like [gcc](https://gcc.gnu.org/) or
     [clang](https://clang.llvm.org/))
--   [Python](https://www.python.org/) (optional, for some tests)
--   GNU [flex](https://github.com/westes/flex) and
-    [bison](https://www.gnu.org/software/bison/), for the Yosys dependency
-    (optional)
```

Comment:
flex and byson are required for building the yosys lib as well, right? if so, mark as `optional, for some tests`? (it might be possible to only require them for the verilog running tests by making yosys an additional dep instead of bundling in the test utilities)